nginx.conf.sample

```diff
@@ -83,6 +83,12 @@ location /static/ {
     if ($MAGE_MODE = "production") {
         expires max;
     }
+    
+    # Remove signature of the static files that is used to overcome the browser cache
+    location ~ ^/static/version {
```

Comment:
Why limit to a `location` block and not always strip version ? Then the code would become:

```
rewrite ^/static/version\d*/(.*)$ /static/$2;
```